pkg/sti/script/installer.go

```diff
@@ -108,6 +109,20 @@ func (s *handler) download(scripts []api.Script, workingDir string) (bool, error
 		}
 	}
 
+	// check for the scripts inside the sources
+	scriptsDir := filepath.Join(workingDir, "/upload/src/.sti/bin")
+	if s.fs.Exists(scriptsDir) {
+		for _, script := range scripts {
+			file := filepath.Join(scriptsDir, string(script))
+			if s.fs.Exists(file) {
+				downloads[script] <- true
+			} else {
+				errs[script] <- errors.NewScriptDownloadError(script, nil)
```

Comment:
Yeah, you're right, although looking at further logic, it's the fact of being downloaded or not (https://github.com/openshift/source-to-image/blob/master/pkg/sti/script/installer.go#L135) decides about the final error. But I'll remove the `else` part as it's not needed there.